src/server/services/training.service.ts

```diff
@@ -151,6 +183,16 @@ export const createTrainingRequest = async ({
 
   // console.log(response);
 
+  if (!response.ok) {
+    await createBuzzTransaction({
+      fromAccountId: 0,
+      toAccountId: userId,
+      amount: price,
+      type: TransactionType.Refund,
+      description: 'Refunding due to an error submitting the training job',
+    });
+  }
```

Comment:
Instead of checking here, we can do an `else` in line 203 and charge the buzz
```suggestion
```
Then in line 200-203
```diff
// line 200

if (!response.ok) {
  // same thing that we have...
+} else {
+ // charge buzz
+ await createBuzzTransaction({
+   fromAccountId: userId,
+   toAccountId: 0,
+   amount: price,
+   type: TransactionType.Training,
+ });
}
```

Reply:
^ see above